examples/receipt.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+# Print a fake receipt
+
+import re, textwrap
+from datetime import datetime
+from escpos import printer
+
+items = [
+    {"sku": 666, "desc": "Dragon Tears", "price": 5.6},
+    {"sku": 661, "desc": "Lizard Tongue", "price": 7.77},
+    {"sku": 911, "desc": "Bat Wing", "price": 13.7},
+    {"sku": 42, "desc": "Towel", "price": 19.99},
+]
+tax_percent = 0.05
+header = {"sku": "SKU", "desc": "Product Description", "price": "Price"}
+header_format = "{sku:<6} {desc:<33} {price:<6}"
+item_format = "{sku:>06} {desc:<33} {price:>6.2f}"
+subtotals_format = "{dummy:6} {desc:>33} {price:>6.2f}"
+
+# As good as time as any
+timestamp = datetime(2012, 12, 21, 11, 59, 59)
+served_by = "Imruryg the Brave"
+
+address = """57 Dandelion Tower Drive
+Glimmerhollow, TQ, 981-PPU
+Phone: +1-403-555-2106"""
+
+disclaimer = """For entertainment purposes only.
+Do not use for summoning demons and/or conjuration of spirits.
+Magick should only be perfomed by trained professionals.
+All sales are final. No refunds or exchanges for enchanted items.
+Hocus Pocus will not be responsible for any damages, injuries, or
+losses that occur while using or misusing these items.
+Always check local bylaws and regulations before invoking any spells.
+"""
+
+# Font "b" on my TM-P80-clone printer can squeeze 64 characters per line
+disclaimer_width = 64
+
+recipt_barcode = "1234567890"
+
+
+# Justify-text on left AND right sides by padding spaces,
+# code by: Georgina Skibinski https://stackoverflow.com/a/66087666
+def justify(txt: str, width: int) -> str:
+    prev_txt = txt
+    while (l := width - len(txt)) > 0:
+        txt = re.sub(r"(\s+)", r"\1 ", txt, count=l)
+        if txt == prev_txt:
+            break
+    return txt.rjust(width)
```

Comment:
I really like this function.
I think we must add it to python-escpos as a private method that other methods like `block_text`, `software_columns` and the likes can make optional use of it.

What do you think @patkan @agordon ?

Reply:
Sounds good, feel free to create a change request :-)